_team/Eto.md

```diff
@@ -0,0 +1,14 @@
+---
+firstname: "Ethan"
+lastname: "To"
+group: "Member"
+title: "Developer"
+img: "me.jpg"

```

Comment:
Do you mind renaming this and the file to a more descriptive name such as `"eto.jpg"`? It will help with name collisions in the future!